internal/schedule_client.go

```diff
@@ -281,6 +281,23 @@ type (
 		//
 		// Deprecated - This is only for update of older search attributes. This may be removed in a future version.
 		UntypedSearchAttributes map[string]*commonpb.Payload
+
+		// Summary - Single-line summary for this workflow execution that will appear in UI/CLI. This can be in
+		// single-line Temporal markdown format.
```

Comment:
None of these strings that reference this markdown format have info about what it is. Probably because it doesn't exist yet, but, if it does somewhere, link would be good.

Reply:
Yeah, as this concept is stabilized (i.e. non-experimental) and we have something at docs.temporal.io, would definitely want to link it here.